Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ponylang/ponyc:release
+
+WORKDIR /src/changelog-tool
+
+COPY Makefile LICENSE bundle.json /src/changelog-tool/
+COPY *.pony /src/changelog-tool/
+
+RUN make \
+  && make install \
+  && make clean
+
+WORKDIR /src/main
```

Comment:
This should be `/src/changelog-tool` I think.

Reply:
should it?

why is it /src/main rather than /src/ponyc in the ponyc Dockerfile?

Reply:
Ah, it's that way in the `ponyc` Dockerfile because it expects you to run by mounting your source code in that directory, so that when you run `ponyc` it will be in the right directory already.

So in this case I guess the best idea would be to just remove this line?

Or are you expecting the invoker to mount the changelog into this directory?

Reply:
I wasn't expecting that. It's just my lack of Docker knowledge. Anything else that should be removed?